pkg/object/grpcserver/spec.go

```diff
@@ -63,15 +64,15 @@ type (
 		IPFilter   *ipfilter.Spec `json:"ipFilter,omitempty" jsonschema:"omitempty"`
 		Host       string         `json:"host" jsonschema:"omitempty"`
 		HostRegexp string         `json:"hostRegexp" jsonschema:"omitempty,format=regexp"`
-		Paths      []*Path        `json:"paths" jsonschema:"omitempty"`
+		Methods    []*Method      `json:"methods" jsonschema:"omitempty"`
 	}
 
-	// Path is second level entry of router.
-	Path struct {
+	// Method is second level entry of router.
+	Method struct {
 		IPFilter       *ipfilter.Spec `json:"ipFilter,omitempty" jsonschema:"omitempty"`
-		Path           string         `json:"path,omitempty" jsonschema:"omitempty,pattern=^/"`
```

Comment:
Why delete `path`, am I missing something?

Reply:
this is the grpc server, not deleting `path` but renaming it to `method`.